### ISSUE TITLE
Build error: "Unable to find modules for some sources"

### ISSUE THREAD

**Issue by lfwells** [dart-browser/universal_html#10]
Hi all, I am getting the following error when importing this package for flutter web. Am I missing something or am I completely off-base?

```Unable to find modules for some sources, this is usually the result of either a
bad import, a missing dependency in a package (or possibly a dev_dependency
needs to move to a real dependency), or a build failure (if importing a
generated file).

Please check the following imports:

import 'package:universal_html/html.dart';
```

I have the package under dependencies in pubspec.yaml:
`universal_html: ^1.1.10` (it is not under dev_dependencies

I am on:
`Flutter 1.13.8-pre.58 • channel master • https://github.com/flutter/flutter.git
Framework • revision fabf4e3d0d (8 hours ago) • 2020-01-08 15:28:02 -0800
Engine • revision 46adf73d51
Tools • Dart 2.8.0 (build 2.8.0-dev.0.0 c0ca187f26)`

**Comment by terrier989** [comment]
Does `flutter pub cache repair` solve the issue?

**Comment by lfwells** [comment]
I returned to this issue after doing some other stuff, and during the process of that I did a `flutter clean` (just that, not what you wrote). When I went to try your suggestion, it was already building no problems. I think it may have just been my packages cache playing up (hence your suggestion). Rookie flutter error, not an issue with this plugin.

Also, confirming this plugin saved my bacon not being able to use `dart:html` on non-web platforms. I wanted my flutter web app to have a pointer mouse cursor for buttons (obviously not needed on Android/ios), but to use the same codebase for Android iOS. Mission completed!
